addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/handler/YamahaBridgeHandler.java

```diff
@@ -139,45 +150,36 @@ private void refreshFromState(ChannelUID channelUID) {
      *            initiate a refresh.
      */
     private void setupRefreshTimer(int initialWaitTime) {
-        BigDecimal intervalConfig = (BigDecimal) thing.getConfiguration()
-                .get(YamahaReceiverBindingConstants.CONFIG_REFRESH);
-        if (intervalConfig != null && intervalConfig.intValue() != refrehInterval) {
-            refrehInterval = intervalConfig.intValue();
+        BigDecimal intervalConfig = (BigDecimal) thing.getConfiguration().get(CONFIG_REFRESH);
+        if (intervalConfig != null && intervalConfig.intValue() != settings.getRefreshInterval()) {
+            settings.setRefreshInterval(intervalConfig.intValue());
         }
 
         if (refreshTimer != null) {
             refreshTimer.cancel(false);
         }
-        refreshTimer = scheduler.scheduleWithFixedDelay(() -> updateAllZoneInformation(), initialWaitTime,
-                refrehInterval, TimeUnit.SECONDS);
+        refreshTimer = scheduler.scheduleWithFixedDelay(() -> updateAllZoneInformation(), initialWaitTime, settings.getRefreshInterval(), TimeUnit.SECONDS);
```

Comment:
The line break limit of this commit is different to how my eclipse was configured through the oomph setup. Does the PR conforms to the OH style guidelines?

Reply:
I am using IntelliJ. The project passed style checks in maven/Jenkins. Wouldn't this be catched if style would be violated?